calculator.py

```diff
@@ -0,0 +1,27 @@
+operations_count = 0
+
+def main():
+    ask_again = True
+    while(ask_again):
+        a = input("Enter the numerator: ")
+        b = input("Enter the denominator: ")
+        result = perform_division(a,b)
+        print(result)
+        ask_again = input("Do you want to perform another operation? Enter yes or no: ")
+        if(ask_again == 'yes'):
+            ask_again = True
+        else:
+            ask_again = False
+            print("You performed " + str(operations_count) + " operations, bye!")
+
+
+def perform_division(a,b):
+    global operations_count
+    try:
+        operations_count += 1
+        return int(a)/int(b)
+    except Exception as e:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Try, Except, Pass detected. https://bandit.readthedocs.io/en/latest/plugins/b110_try_except_pass.html